src/Symfony/Component/Messenger/Command/ConsumeMessagesCommand.php

```diff
@@ -77,6 +77,7 @@ protected function configure(): void
                 new InputOption('queues', null, InputOption::VALUE_REQUIRED | InputOption::VALUE_IS_ARRAY, 'Limit receivers to only consume from the specified queues'),
                 new InputOption('no-reset', null, InputOption::VALUE_NONE, 'Do not reset container services after each message'),
                 new InputOption('all', null, InputOption::VALUE_NONE, 'Consume messages from all receivers'),
+                new InputOption('exclude-receivers', 'er', InputOption::VALUE_REQUIRED | InputOption::VALUE_IS_ARRAY, 'Exclude specific receivers from consumption (can only be used with --all)'),
```

Comment:
Sine the parameter `queues` allows to specify only certains queues to be consumed, I thounk `exclude-queues` would makes more sense:
```suggestion
                new InputOption('exclude-queues', 'er', InputOption::VALUE_REQUIRED | InputOption::VALUE_IS_ARRAY, 'Exclude specific receivers from consumption (can only be used with --all)'),
```

Reply:
please update the PR description appropriately